src/tests/issues.targets

```diff
@@ -280,6 +280,9 @@
         <ExcludeList Include="$(XunitTestBinBase)/JIT/Regression/JitBlue/Runtime_31615/Runtime_31615/*">
             <Issue>https://github.com/dotnet/runtime/issues/79170</Issue>
         </ExcludeList>
+        <ExcludeList Include="$(XunitTestBinBase)/JIT/Regression/JitBlue/Runtime_63354/Runtime_63354/*">
```

Comment:
The file is `JIT\Regression\JitBlue\Runtime_63354\Runtime_63354.csproj`, not sure how it would work with this wildcard. I see that they are used at other places too. But shouldn't it be `$(XunitTestBinBase)/JIT/Regression/JitBlue/Runtime_63354/**`?

Reply:
There are only two files on that directory (src\tests\JIT\Regression\JitBlue\Runtime_63354), we could add the ** to be sure anyway